src/fs/fs_common.h

```diff
@@ -23,14 +23,31 @@
 #ifndef SRC_FS_FS_COMMON_H_
 #define SRC_FS_FS_COMMON_H_
 
+#include <string>
+
 namespace curve {
 namespace fs {
 
 enum class FileSystemType {
     // SFS,
     EXT4,
+    PFS,
+    UNKOWN,
 };
 
+inline static FileSystemType StringToFileSystemType(
```

Comment:
what's the purpose of `static` here? I think only `inline` is necessary.

Reply:
done